### PR TITLE
msamprz/elevio-lang-fix

### PR DESCRIPTION
**Pull request by msamprz**

### CODE REVIEW

File: src/javascript/_common/base/elevio.js [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
+const localize        = require('../localize').localize;
+const createElement   = require('../utility').createElement;
 
 const Elevio = (() => {
     const init = () => {
         if (!window._elev) return; // eslint-disable-line no-underscore-dangle
         window._elev.on('load', (elev) => { // eslint-disable-line no-underscore-dangle
-            const available_elev_languages = ['es', 'id', 'pt', 'ru'];
-            const current_language         = getLanguage().toLowerCase();
+            const available_elev_languages = ['de', 'fr', 'id', 'it', 'po', 'pt', 'ru', 'vi'];
```
Comment (negarn) [inline src/javascript/_common/base/elevio.js]: we shouldn't be adding more languages, our custom translation is not fully ready from elevio

File: src/javascript/_common/base/elevio.js [hunk 2]
```diff
@@ -1,18 +1,20 @@
...
+            const current_language         = languageFromUrl().toLowerCase();
```
Comment (negarn) [inline src/javascript/_common/base/elevio.js]: getLanguage also calls languageFromUrl, what was the issue that this is solving?

Comment (msamprz) [reply src/javascript/_common/base/elevio.js]: noted and addressed.

File: src/javascript/_common/base/elevio.js (hunk 1 above)
Comment (msamprz) [reply src/javascript/_common/base/elevio.js]: noted and addressed.

**Pull request merged**